src/PowerShellEditorServices/Services/PowerShell/Host/PowerShellFactory.cs

```diff
@@ -1,120 +0,0 @@
-﻿using System;
```

Comment:
This now belongs to the host, since it needed enough internal information from it anyway